src/localStorage.js

```diff
@@ -0,0 +1,12 @@
+export const getLocalStorage = (toDo) => {
+  if (localStorage.getItem('toDo')) {
+    toDo.list = JSON.parse(localStorage.getItem('toDo'));
+    for (let i = 0; toDo.list.length > i; i += 1) {
+      toDo.createToDo(toDo.list[i]);
+    }
```

Comment:
Kindly consider using for each loop instead of for loop to meet ES6 best practices